__init__.py

```diff
@@ -46,8 +46,8 @@ class ApiAddonPreferences(AddonPreferences):
     )
 
     freesound_download_folderpath : StringProperty(
-        name = "Download folder path",
-        description = "Folder path where freesound downloads are stored",
+        name = "Common folder path for download",
+        description = "Common folder path where freesound downloads are stored",
         subtype = 'DIR_PATH',
         default = os.path.join(bpy.utils.user_resource('DATAFILES'), "freesound_downloads")
```

Comment:
done, for the record, I think it could be better practice to import the entire library for better readability, specially for open source code (a contributor does not have to know about os module or check in a code editor to get what is "join", "os.path.join" is more explicit (and the method is used only once here, no real gain in characters)
just my point of view about it.